Feature/timeline param

agnlez opened the pull request.

agnlez committed:
pull

agnlez committed:
Merge branch 'feature/share' into develop

agnlez committed:
Merge branch 'develop' of github.com:Vizzuality/disarm into feature/timeline-param

agnlez committed:
set timelineDate param and setup

dhakelila commented on src/app.jsx:
```diff
@@ -63,16 +64,19 @@ class App extends React.Component {
       },
-      layers: []
+      layers: [],
+      timelineDate: new Date('2012-01-01')
```
Use UTC, please. For consistency and to avoid issues...

dhakelila commented on src/app.jsx:
```diff
@@ -120,12 +124,14 @@ class App extends React.Component {
+    } else {
+      timelineParams.cursorPosition = this.state.timelineDate;
+    }
 
```
We already have a function where we retrieve all router params and set them as app state. You'd only need to take that param from app state.

dhakelila commented on src/app.jsx:
```diff
@@ -63,16 +64,19 @@ class App extends React.Component {
+      this.updateRouter(params);
+    } ,this)
   }
 
```
We already call an app method from the timeline to update router. The only thing you need to update aside of this is the app state. (already on that function `updateTimelineDates(){}`)

agnlez committed:
timeline date changes (see desc. for more info)

- First update date triggered by own function
- default date is in UTC now
- removed check params for timelineDate

dhakelila committed:
avoid inecesary function

dhakelila committed:
setting timeline cursor at the end of the timeline
